src/core/pages/PatientsList/PatientEdit.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { TextInput, DateInput, RadioButtonGroupInput } from "react-admin";
+
+import { withStyles } from '@material-ui/core/styles';
+
+import EditTemplate from "../../common/ResourseTemplates/EditTemplate";
+
+const styles = {
```

Comment:
these are the same styles as in the PatientCreate, you should extract and share them

Reply:
Done: I created separate component for form inputs and extract choices there. Styles are located in this component.